src/user-service/__tests__/userRoutes.test.ts

```diff
@@ -1,5 +1,5 @@
 import request from 'supertest';
-import app from '../app';
+import app from '../../app';
 import UserService from '../userService';
 
 jest.mock('../userService');
```

Comment:
Unit tests are used for business logic cases. It's usually needed when there is a complex business logic that needs to be tested, which is not the case here.

 To test the flow, it's gonna be better to have e2e tests, like a test to create, then verify, then get the user.